core/src/main/java/org/pentaho/di/core/extension/KettleExtensionPoint.java

```diff
@@ -131,7 +131,9 @@ public enum KettleExtensionPoint {
     AfterDeleteRepositoryObject( "AfterDeleteRepositoryObject",
       "After an object has been deleted from the repository" ),
 
-    TransformationCreateNew( "TransformationCreateNew", "Create a New Empty Transformation in Spoon" );
+   TransformationCreateNew( "TransformationCreateNew", "Create a New Empty Transformation in Spoon" ),
+   TransSharedObjectsLoaded( "TransSharedObjectsLoaded", "After the shared objects are loaded in Transformations." ),
+   JobSharedObjectsLoaded( "JobSharedObjectsLoaded", "After the shared objects are loaded in Job." );
```

Comment:
These are actually while the Meta is loaded, not the Job itself. There are other JobMeta/TransMeta ExtensionPoints. Maybe name these like those instead?

Reply:
Right, but these aren't being called when the Transformation or Job are being loaded, they're being called while the TransMeta and JobMeta are being loaded. I'm saying these should be named something like `TransformationMetaSharedObjectsLoaded` just like `TransformationMetaLoaded` (same with jobs).

Reply:
We can always add more context in the name but then it become too long.   I thought the two key things to add to the name is job/trans + when does it get called

Reply:
"when does it get called" is not accurate here. All the "TransformationFoo" KettleExtensionPoints are called in Trans.java or when running a transformation. All the ones in TransMeta.java are "TransformationMetaFoo"